src/lib.rs

```diff
@@ -25,6 +27,34 @@ impl Script {
 impl_from_core_type!(Script, BitcoinScriptBuf);
 impl_from_ffi_type!(Script, BitcoinScriptBuf);
 
+#[derive(Debug, Clone)]
+pub struct TxOut {
+    pub value: Arc<Amount>,
+    pub script_pubkey: Arc<Script>,
+}
+
+impl From<&BitcoinTxOut> for TxOut {
```

Comment:
I think we will want to use an owned `BitcoinTxOut`

Reply:
will change this rn